### DISCUSSION TITLE
Can't get the image with black borders in the Gallery View on Mobile

### DISCUSSION THREAD

**Discussion by stbs01** [biati-digital/glightbox#505] [Q&A]
Hi!

I am using a https://github.com/paulmassen/grav-theme-hyde Grav Theme (v0.2.2) with a Premium Lightbox Gallery Plugin (v1.0.6) on a Linux VPS Server. Now I have the following problem with the Lightbox Gallery Plugin:
<img width="628" alt="Screenshot 2024-07-02 at 19 40 23" src="https://github.com/biati-digital/glightbox/assets/174446128/fb1bf638-706a-4125-a185-c332dbcf3b0b">

I want to have around every image on my website a black stripe, like here as you see it from PC:

(is is kind of OK-ish here, because the stripe on the bottom should be the same width as on the top - any ideas how to do it?)

Here is how I see images from my mobile devices:
<img width="227" alt="Screenshot 2024-07-02 at 19 40 40" src="https://github.com/biati-digital/glightbox/assets/174446128/085580b9-c25e-4f74-80e9-13af54c92095">
<img width="624" alt="Screenshot 2024-07-02 at 19 40 59" src="https://github.com/biati-digital/glightbox/assets/174446128/9fab8141-1f01-4d69-8086-bdb0915b3574">
<img width="624" alt="Screenshot 2024-07-02 at 19 41 13" src="https://github.com/biati-digital/glightbox/assets/174446128/d9b06614-258e-4027-bcc3-5550eda08a6c">

So - no image titles, no borders on top and bottom.

If you go to http://draft.teimur-bek-safiuli.com/works from your smartphone/tablet, you will see it.

Thanks for your help!

Best,
Teimur

studio@teimur-bek-safiuli.com

P.S. I have a pretty experienced guy who helps me to build my artist portfolio website, but here he cant help me anymore, that's why I need your help 🙏🏼

**Comment by biati-digital** [comment]
Please do not send us messages through our website asking for support. Your messages were moved to the discussion section because they involve custom modifications. You need to be really patient as our community members volunteer their time to assist so it can take a long time until you get a response. Keep in mind that GLightbox is a free library developed in our spare time, and we do not offer support or custom modifications.

If you are willing to pay for the modifications you need, I can recommend someone who is very knowledgeable about the library.

**Reply by stbs01** [reply]
@biati-digital Thanks for your answer!
"If you are willing to pay for the modifications you need, I can recommend someone who is very knowledgeable about the library."
Please do recommend! studio@teimur-bek-safiuli.com

**Reply by stbs01** [reply]
@biati-digital I did not receive your referral yet. Thanks in advance!

**Reply by stbs01** [reply]
@biati-digital Would you send me the reference please?

**Reply by biati-digital** [reply]
Hi @stbs01 Sorry for the delay. I’ve been unwell and needed some time to rest. I have contacted the person to check availability and am waiting for a response. I’ll keep you posted.

**Reply by stbs01** [reply]
> Hi @stbs01 Sorry for the delay. I’ve been unwell and needed some time to rest. I have contacted the person to check availability and am waiting for a response. I’ll keep you posted.

Thanks for your reply. I do need to talk to the developer of the Premium Gallery Plugin or to an expert in this area, because this plugin has some very bad bugs, which cannot be fixed by my webmaster.

**Reply by stbs01** [reply]
And I cant do much since the Premium Gallery Plugin (https://getgrav.org/premium/lightbox-gallery/docs) was built actually by you guys at @biati-digital : "This plugin is built on top of the amazing GLightbox library by biati digital". And if the Lightbox Gallery Premium doestn work, who can fix it? Only you @biati-digital as I understand. Because I already put it as issues at https://github.com/getgrav/grav-premium-issues/issues, and @rhukster answered me: "This is a question for the author of GLitghtbox. As i said in a previous issue you posted, i didn't author the lightbox script, just the integration of the script into Grav." So I urgently need support from the specialist who actually did this GLightbox library..... So far on the context

**Reply by biati-digital** [reply]
> And I cant do much since the Premium Gallery Plugin (https://getgrav.org/premium/lightbox-gallery/docs) was built actually by you guys at @biati-digital 

That's not true, we did not create that plugin and we are not affiliated in any way, there's many developers taking our free work an using it on their "Premium" plugins. We only created this library and we are not selling it anywhere.

So, that company is taking our free work and charging for it, yet they can’t even check the library code to assist their customers? This is an open-source library that we created in our free time as a side project. We do not offer any support for it and do not earn anything from it.

I created the library but don’t have the time to work on this at the moment. You’ll need to wait until I get a reply from another developer to check their availability.

**Reply by gingerchew** [reply]
@stbs01 Hey there, I'm the mentioned developer, I just sent you an email at the address you commented above.
